### ISSUE TITLE
Bug: Broken Google OAuth login- RECAPTCHA_VERIFY_FAILED on platform.deepseek.com

### ISSUE THREAD

**Issue by rohit1kumar** [deepseek-ai/DeepSeek-V3#170]
**Describe the bug**
Receiving RECAPTCHA_VERIFY_FAILED on platfrom.deepseek.com while trying to log in with Google for API access, but I am able to log in successfully with the same email on chat.deepseek.com. 

**To Reproduce**
Go to [deepseek.com](https://www.deepseek.com) > Access API > Google login

**Screenshots**
![image](https://github.com/user-attachments/assets/e9d1d887-4f1a-458b-9900-91c6122dc3c4)

**Comment by Axenide** [comment]
Same here.

**Comment by dejanr** [comment]
Same here

**Comment by spullara** [comment]
Same

**Comment by schacki** [comment]
Same here

**Comment by mowentian** [comment]
Received. We are working on resolving the issue...

**Comment by brendanp** [comment]
same

**Comment by weicfd** [comment]
Case reproduced.  
Triggered by oversea users using web ad blocking extensions (Adblock etc.) trying to fetch device info.
Fixed. Please try to login again. Refresh page is optional.

**Comment by rohit1kumar** [comment]
@weicfd working thanks

**Comment by GEMCorp** [comment]
I don't have any ad block but not able to sign in to chat.deepseek.com on a new chrome installation
<img width="784" alt="image" src="https://github.com/user-attachments/assets/e1022570-6ede-4325-b5e7-ea54b3323b73" />

**Comment by Ki3ro** [comment]
I have the same problem as @GEMCorp  . Interestingly when trying to log in when I click on "Start Now" I cannot log in. However, when I log in via "Access API" I can log in. Currently I cannot access chat.deepseek.com
![image](https://github.com/user-attachments/assets/c34bb232-bc01-414f-8f53-ac743507f236)

**Comment by dejanr** [comment]
Yes, it worked yesterday but today i get `{"detail":"Not Found"}`, hope they fix it again.

**Comment by drcybermo** [comment]
Same problem in Sweden. Same message when login with google account. Annoying! Plz fix asap!

**Comment by tasuki** [comment]
There's a [(probably temporary) workaround](https://github.com/deepseek-ai/DeepSeek-V3/issues/220#issuecomment-2572948617) here.

**Comment by ilsalgo** [comment]
same here

**Comment by leysin1** [comment]
same problem as @GEMCorp, trying to connect from germany

**Comment by wen080392** [comment]
> **Descreva o bug** Recebendo RECAPTCHA_VERIFY_FAILED em platfrom.deepseek.com ao tentar efetuar login com o Google para acesso à API, mas consigo efetuar login com sucesso com o mesmo e-mail em chat.deepseek.com.
> 
> **Para reproduzir,** acesse [deepseek.com](https://www.deepseek.com) > API de acesso > Login do Google
> 
> **Capturas de tela** ![image](https://github.com/user-attachments/assets/e9d1d887-4f1a-458b-9900-91c6122dc3c4)

Eu acho que isso não é normal eu acho que é porque uma inteligência artificial da China eles tão querendo barra o crescimento dela que já tá passando o chat gpt

**Comment by userse31** [comment]
Now, I'm not saying the Americans are blacklisting Deepseek.

Except that is EXACTLY what I'm saying, and that this is a big old conspiracy by the US capitalists! Fricken... 2nd Cold War.....

**Comment by Darrylwin** [comment]
it's becoming very worrying
